tftb/processing/linear.py

```diff
@@ -11,6 +11,9 @@
 """
 
 import numpy as np
+import scipy.signal
+from scipy.signal.windows import hamming
```

Comment:
Same as above. The two different imports seem redundant.

Reply:
Ok, I will revert this.